plenum/server/view_change/view_changer.py

```diff
@@ -477,9 +478,15 @@ def do_view_change_if_possible(self, view_no):
             logger.info(whyNot)
         return can
 
-    def _start_view_change_if_possible(self, view_no) -> bool:
+    def _qourum_is_reached(self, count, is_initial_propagate_primary):
+        if is_initial_propagate_primary:
+            return self.quorums.propagate_primary.is_reached(count)
+        else:
+            return self.quorums.view_change_done.is_reached(count)
+
+    def _start_view_change_if_possible(self, view_no, is_initial_propagate_primary=False) -> bool:
         ind_count = len(self._next_view_indications[view_no])
-        if self.quorums.propagate_primary.is_reached(ind_count):
+        if self._qourum_is_reached(ind_count, is_initial_propagate_primary=is_initial_propagate_primary):
```

Comment:
This is the quorum for starting of a view change for Primary propagation.
Please also have a look at `quorum` property in ViewChanger